frontend/src/routes.js

```diff
@@ -112,6 +113,7 @@ const routes = (
     {/*Codes*/}
     <Route path={`${process.env.PUBLIC_URL}/codes`} element={<PrivateRoute element={Codes}/>}/>
     <Route path={`${process.env.PUBLIC_URL}/code/new`} element={<PrivateRoute element={AddEditCode}/>}/>
+      <Route path={`${process.env.PUBLIC_URL}/code/:uri/view`} element={<PrivateRoute element={Code}/>}/>
```

Comment:
Please make sure the file is well formatted(reformatted). On WebStorm, you can use option-command-l to achieve this.